AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -625,7 +625,7 @@ private static void checkAndStartNotificationsModules() {
 
   private static void startHelper(
       String userId, final Map<String, ?> attributes, final boolean isBackground) {
-    LeanplumEventDataManager.init(context);
+    LeanplumEventDataManager.sharedInstance();
```

Comment:
Can you please explain why initialization wasnt happening? I see init here correct

Reply:
As you can see, initialization happens in `startHelper` which is actually causing the issues when app is not running (i.e push notification received).